src/App.js

```diff
@@ -24,11 +31,12 @@ const App = () => (
           <Route path="/signup" exact component={Signup} />
           <Route path="/reset-password" exact component={resetPasswordView} />
           <Route path="/forgot-password" exact component={forgotPasswordView} />
+          <Route path="/signout" exact component={Signout} />
```

Comment:
There is no need for the `route` of the signout. I would suggest writing a function that would perform the signout rather than having a route to be called while signing out.